# requested command
HybridImpedanceCommandTrajForceMsg command
---
# current command
HybridImpedanceCommandTrajForceMsg command